sys/include/net/gnrc/pktbuf.h

```diff
@@ -285,6 +285,11 @@ gnrc_pktsnip_t *gnrc_pktbuf_reverse_snips(gnrc_pktsnip_t *pkt);
  *
  *        The original snip is keeped as is except `users` decremented.
  *
+ * @deprecated  This function breaks the abstraction of `gnrc_pktbuf` and its
+ *              only user within the RIOT code base `gnrc_ipv6_ext` was reworked
+ *              so it isn't needed anymore.
```

Comment:
If it should be merged before https://github.com/RIOT-OS/RIOT/pull/10234 I would prefer saying something as "is being reworked to not need it anymore".

To use a verb in the past, the fix should be merged before the deprecation :)

Reply:
Welp, but that would require another grammar fix as soon as #10234 is merged (which we might forget).

Reply:
Ok. Reworded it. I'll write a reminder into #10234.